### PR TITLE
issues #2, #3, #4, #5, #6, #7

### PR DESCRIPTION
**Pull request by vicfriedman**

### CODE REVIEW

**Comment by AnnJohn** [comment]
Reassigned to @ipc103

File: README.md [hunk 1]
```diff
@@ -67,7 +47,7 @@ Math.floor(4.00001);                     // Returns 4
 Math.floor(4.0000000000000001); === true // safer!
```
Comment (ipc103) [inline README.md]: should this be `Math.floor(4.0000000000000001) === 4; // safer!`?

File: README.md [hunk 2]
```diff
@@ -13,49 +19,23 @@
...
-
-```javascript
-var greeting = Symbol("hello");
-typeof sym; // Returns "symbol"
-```
-
-#### 1. Numbers
+### Numbers
 
 Unlike most programming languages JS uses floats by default. And creating literal numbers uses the Number Constructor. New numbers can optionally have decimal points with trailing zeros, but keep in mind they are floats either way.
```
Comment (ipc103) [inline README.md]: I'm confused about the Number Constructor - can "And creating literal numbers uses the Number Constructor." be removed?

**Comment by ipc103** [comment]
made some minor edits and comments - assigning back to @vicfriedman

File: README.md (hunk 1 above)
Comment (vicfriedman) [inline README.md]: YES

File: README.md (hunk 2 above)
Comment (vicfriedman) [inline README.md]: removing.

**Pull request merged**